views/index.html

```diff
@@ -156,7 +156,7 @@ <h2 class="call-to-action">Let's Talk!</h2>
 
         <fieldset class="pure-group">
           <label for="budget">Budget:</label>
-          <select id="budget" name="budget" required>
+          <select size="5" id="budget" name="budget" required>
```

Comment:
Out of curiosity, why would you choose 5 instead of 3 seeing as there are only 3 visible options?